Base/usr/share/man/man8/useradd.md

```diff
@@ -0,0 +1,50 @@
+## Name
+
+useradd - add a new user to the system password file
+
+## Synopsis
+
+```**sh
+# useradd [options] LOGIN
+```
+
+## Description
+
+This program uses adds a new user to the system.
+
+By default, the user will be added to the **users** group (which has a GID of 100).
+
+This program must be run as root.
+
+## Options
+
+* `-u`, `--uid` _uid_: The user identifier for the new user. If not specified, an unused UID above `1000` will be auto-generated.
+* `-g`, `--gid` _gid_: The group identifier for the new user. If not specified, it will default to 100 (the **users** group).
+* `-s`, `--shell` _shell_: The shell binary for this login. The default is `/bin/Shell`. 
+* `-m`, `--create-home`: Create a home directory for this new user at `/home/username`.
+* `-d`, `--home-dir` _path_: Set the home directory for this user to path).
+* `-n`, `--gecos` _general-info_: GECOS information about this login. See [Wikipedia](https://en.wikipedia.org/wiki/Gecos_field) for more information.
+
+## Exit Values
+
+* 0 - Success
+* 1 - Couldn't update the password file
+* 3 - Invalid argument to option
+* 4 - UID already in use
+* 12 - Couldn't create home directory
+
+## Files
+
+* `/etc/passwd` - new user information (such as UID and GID) is appended to this file.
+* `/home/` - user home directroy is created here if the `-m` flag is specified.
+
+## Examples
+
+```sh
+# useradd -u 300 -m kling
+# useradd -m -u 400 --gecos 200 -n "Sergey Bugaev" bugaevc
```

Comment:
Wait, does `-n` do something different from `--gecos`? I though it's the same option.

Reply:
whoops that's --gid